README.md

```diff
@@ -0,0 +1,79 @@
+WSO2 Application Manager (Carbon Components)
```

Comment:
Can we rename WSO2 Application Manager references to "WSO2 App Manager" ? We initially had name as Application Manager, but later we renamed as App Manager.